src/locale/i18n.ts

```diff
@@ -91,6 +92,11 @@ export async function dynamicActivate(locale: AppLanguage) {
       await import('@formatjs/intl-pluralrules/locale-data/pt')
       break
     }
+    case AppLanguage.ru: {
+      i18n.loadAndActivate({locale, messages: messagesRu})
+      await import('@formatjs/intl-pluralrules/locale-data/ru')
```

Comment:
```suggestion
      await Promise.all([
        import('@formatjs/intl-pluralrules/locale-data/ru'),
        import('@formatjs/intl-numberformat/locale-data/ru'),
      ])
```
I think this should be updated as a result of #5128

Reply:
Thanks for the clarification! Corrected!